Makefile

```diff
@@ -167,6 +167,23 @@ integration-test: #End to end test DI project - mandatory: PROFILE, TAGS=[comple
 		-e CR_FIFO_DLQ=$(TF_VAR_cr_fifo_dlq_handler_lambda_name) \
 		"
 
+create-dentist-reports:
+	make -s docker-run-tools \
+	IMAGE=$$(make _docker-get-reg)/tester:latest \
+	CMD="python comparison_reporting/run_dentist_reports.py" \
+	DIR=./application \
+	ARGS=" \
+		-e PYTHONPATH="." \
+		-e DB_SERVER=$(DB_SERVER) \
```

Comment:
I believe this should be 
-e DB_SERVER=$(DB_SERVER_NAME) \

Reply:
As discussed in call I have changed this to 
`-e DB_SERVER=$$(make -s aws-rds-describe-instance-value DB_INSTANCE=$(DB_SERVER_NAME) KEY_DOT_PATH=Endpoint.Address)`